pytorch_to_returnn/torch/nn/functional.py

```diff
@@ -344,3 +344,16 @@ def norm(input: Tensor,
 
 def norm_except_dim(v: Tensor, pow: int = 2, dim: int = 0) -> Tensor:
   return modules.Norm(p=pow, axes=[i for i in range(v.dim()) if i != dim], keepdims=True)(v)
+
+
+def group_norm(input, num_groups, weight=None, bias=None, eps=1e-5) -> Tensor:
+  if weight is None:
+    assert bias is None
+    module = modules.GroupNorm(num_groups=num_groups, num_channels=0, eps=eps, affine=False)
+  else:
+    assert bias is not None
+    assert weight.shape == bias.shape
+    module = modules.GroupNorm(num_groups=num_groups, num_channels=weight.shape[0], eps=eps)
+    module.register_parameter("weight", weight)
+    module.register_parameter("bias", bias)
```

Comment:
I think this is wrong. I think you cannot (should not) use `register_parameter` for a non-parameter (i.e. regular tensor).
Also, it would do the wrong thing. It is not a parameter here.
You need to also set `affine=False` in this case, and multiply by `weight`, add by `bias`.

Reply:
> You need to also set `affine=False` in this case, and multiply by `weight`, add by `bias`.

True. That's not really supported so far, right? The shapes are different and if I do something like `out = out * weight.view(1, -1, 1)` which should work in PyTorch, the new axes from `view` do not match the axes of `out` in terms of being batch dim/spatial axes in returnn.

Reply:
But that should still work anyway. If not, consider this a bug, and we should fix this.

Reply:
Should we handle that in a separate issue/PR? The default use case of `GroupNorm` works because this `functional.group_norm` is not used there. However, it will currently fail for `Fp32GroupNorm` (see above).

Reply:
No, the default `GroupNorm` must not be used here. In general, we must not use any module with parameters in the functional part.

Reply:
But yes, I think it's easier to discuss and fix the `view` or broadcasting problem (or whatever the problem is) in another PR.

Reply:
Alright!